launch.go

```diff
@@ -352,12 +353,6 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	fmt.Fprintf(emitter, "%s%s\n", blackSprint("Source Dir:     "), w.Src)
 	fmt.Fprintf(emitter, "%s%s\n", blackSprint("Artifacts Dir:  "), w.Artifacts)
 
-	oldJobName := job.Name
-	pr := prNumber(job.Name)
-	if pr != "" {
```

Comment:
Is this if statement no longer necessary?

Reply:
I think it's unnecessary. These are legacy of old workflow era (all pull request jobs are named `main`)

Reply:
👍